cinch/roles/jenkins_master/templates/roles_and_ldap_auth.groovy

```diff
@@ -0,0 +1,177 @@
+import hudson.*;
```

Comment:
80 (79, actually, because Guido is a dork) is only our specification for Python code. Yaml files we permit out to 100, and we had spoken about the lack of need for Groovy/Java to have such strict types of limitations. There were two lines in this file that were above 100 lines, so I've cut them down to under that limit, as we had talked about keeping it in the 100-110 limit for side-by-side diffs.

As for indents: they are consistent throughout this file. Block indents are one tab character, alignment of arguments within hanging indents for line continuation, etc, are all done with space. This gives a clean, manageable, and user-configurable tab stop width while also allowing proper alignment and hanging indents.

One of the beauties of Groovy is that indents are syntactically meaningless. This allows greater flexibility in achieving legibility with proper alignments and such. Can you point specifically to the indents that are bugging you, because this file reads to me exactly how I would expect it to read in Groovy. (And no, I'm not OK with settling on 4 space indents for languages other than Python)